frontend/src/pages/Marketplace.jsx

```diff
@@ -0,0 +1,214 @@
+            <Column width="2-3" horizontalAlignment="center">
+               <TextView text={"Featured App"} heading={2} align={"left"} />
+               <TextView text={"Our favourite usage of the API"} heading={5} align={"left"} />
+               { this.state.featuredApps.map((app, i) => {
+                  return (
+                    <CardView key={"featured-app-"+i} width={"1-1"} style={"emphasis"} link={"/marketplace/" + app.id} addPadding={true}>
+                      <Column width="1-2" horizontalAlignment="center">
+                        <ImageView src={app.logo} width={iconsize} height={iconsize} />
+                        <TextView text={app.name} heading={2} align={"center"} color={"white"}/>
+                        <TextView text={app.description} heading={5} align={"center"} color={"white"}/>
+                      </Column>
+                    </CardView>
+                  );
+               })}
+            </Column>    
+          </Row>
+
+          <Row color="splash-parallax">
+            <Column width="2-3" horizontalAlignment="center">
+               <TextView text={"All Apps"} heading={2} align={"left"} />
+               <TextView text={"Every app made using the API"} heading={5} align={"left"} />
+               { this.state.appsToRender.map((app, i) => {
+                  var margin = "0";
+                  if(i%2 == 0) {margin = "0 2% 0 0"}
+
+                  return (
+                    <CardView key={"all-apps-"+i} width={"1-2"} style={"alternate"} link={"/marketplace/" + app.id} addPadding={true}>
+                      <Column width="9-10" horizontalAlignment="center">
+                        <ImageView src={app.logo} width={iconsize} height={iconsize} />
+                        <TextView text={app.name} heading={2} align={"center"} color={"black"}/>
+                        <TextView text={app.description} heading={5} align={"center"} color={"black"}/>
+                      </Column>
+                    </CardView>
+                  );
+               })}
+            </Column>
+          </Row>
+
+          <Footer />
+
+        </React.Fragment>
+      );
+    }
+}
+
+class AppPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Grab the app that this page is dealing with
+    let app = allApps[this.props.appId];
+    this.state = {
+      "app": app
+    };
+  }
+
+  render () {
+    var iconsize = "100px";
+    var logosize = "150px";
+
+    var screenshotwidth = "216px";
+    var screenshotheight = "384px";
+
+    return (
+      <React.Fragment>
+          <NavBar isScroll={false} />
+
+          <Row height="600px" color="splash-parallax">         
+            <Column width="2-3" horizontalAlignment="center" verticalAlignment="center">
+              <TextView text={"UCL Marketplace"} heading={1} align={"center"}/>
+              <TextView text={"Apps to improve student life at UCL"} heading={2} align={"center"}/>

```

Comment:
Maybe "Apps that use the UCL API" instead? We'd coceivably include apps that help staff, etc., here as well.

Reply:
Changed